src/tools/kdb/pluginlist.hpp

```diff
@@ -35,7 +35,8 @@ class PluginListCommand : public Command
 
 	virtual std::string getLongHelpText () override
 	{
-		return "";
+		return "With -v the output will be sorted by the status of the plugin.\n"
```

Comment:
Usually, we do not use this getLongHelpText anymore. Write it in doc/help/kdb-plugin-list.md instead.

Reply:
Did so in the new commit